datarobot_batch_scoring/main.py

```diff
@@ -213,6 +213,11 @@ def parse_args(argv, standalone=False, deployment_aware=False):
                         'empty name is used if not specified. For binary '
                         'predictions assumes last class in lexical order '
                         'as positive')
+    csv_gr.add_argument('--pred_threshold', type=str,
+                        nargs='?', default=None,
+                        help='Specifies column name for prediction threshold'
```

Comment:
Should have trailng space :)

Reply:
And, as agreed, please fix the docs below to smth like "column will be present in output, if not specified"